cc/crypto/server_encryptor.h

```diff
@@ -38,8 +39,11 @@ namespace oak::crypto {
 // be multiple responses per request and multiple requests per response.
 class ServerEncryptor {
  public:
-  ServerEncryptor(const KeyPair& server_key_pair)
-      : server_key_pair_(server_key_pair),
+  // Constructor for `ServerEncryptor`.
+  // `RecipientContextGenerator` argument is a long-term object containing the private key and
+  // should outlive the per-session `ServerEncryptor` object.
+  ServerEncryptor(RecipientContextGenerator& recipient_context_generator)
```

Comment:
Following on from the above, if that method you call could be made `const`, then you could take ` const&` here.